orb_models/finetune_utilities/experiment.py

```diff
@@ -0,0 +1,128 @@
+"""Experiment utilities."""
+
+import os
+import random
+from collections import defaultdict
+from pathlib import Path
+from typing import Dict, Mapping, TypeVar
+
+import dotenv
+import numpy
+import torch
+import wandb
+from wandb import wandb_run
+
+from orb_models.forcefield import base
+
+T = TypeVar("T")
+
+
+_V = TypeVar("_V", int, float, torch.Tensor)
+
+dotenv.load_dotenv(override=True)
+PROJECT_ROOT: Path = Path(
```

Comment:
Can you remove the concept of PROJECT_ROOT, DATA_ROOT and WANDB_ROOT - these should either use a fixed location or use the default in the case of wandb.